tests/foreman/cli/test_repository.py

```diff
@@ -691,3 +699,162 @@ def test_positive_upload_content(self):
             "Successfully uploaded file '{0}'".format(RPM_TO_UPLOAD),
             result[0]['message'],
         )
+
+
+class RepositoryExportTestCase(CLITestCase):
+    """Tests for exporting a repository via CLI"""
+
+    def setUp(self):
+        """Create a directory for export, configure permissions and satellite
+        settings
+        """
+        self.export_dir = gen_string('alphanumeric')
+
+        # Create a new 'export' directory on the Satellite system
+        result = ssh.command('mkdir /mnt/{0}'.format(self.export_dir))
+        self.assertEqual(result.return_code, 0)
+
+        result = ssh.command(
+            'chown foreman.foreman /mnt/{0}'.format(self.export_dir))
+        self.assertEqual(result.return_code, 0)
+
+        result = ssh.command('ls -Z /mnt/ | grep {0}'.format(self.export_dir))
+        self.assertEqual(result.return_code, 0)
+        self.assertGreaterEqual(len(result.stdout), 1)
+        self.assertIn('unconfined_u:object_r:mnt_t:s0', result.stdout[0])
+
+        # Fix SELinux policy for new directory
+        result = ssh.command(
+            'semanage fcontext -a -t foreman_var_run_t "/mnt/{0}(/.*)?"'
+            .format(self.export_dir)
+        )
+        self.assertEqual(result.return_code, 0)
+
+        result = ssh.command('restorecon -Rv /mnt/{0}'.format(self.export_dir))
+        self.assertEqual(result.return_code, 0)
+
+        # Assert that we have the correct policy
+        result = ssh.command('ls -Z /mnt/ | grep {0}'.format(self.export_dir))
+        self.assertEqual(result.return_code, 0)
+        self.assertGreaterEqual(len(result.stdout), 1)
+        self.assertIn(
+            'unconfined_u:object_r:foreman_var_run_t:s0', result.stdout[0])
+
+        # Update the 'pulp_export_destination' settings to new directory
+        Settings.set({
+            'name': 'pulp_export_destination',
+            'value': '/mnt/{0}'.format(self.export_dir),
+        })
+
+    def tearDown(self):
```

Comment:
It's fine. Executed following code:
```python
class VerifyTearDownClass(CLITestCase):

    def setUp(self):
        super(VerifyTearDownClass, self).setUp()
        self.fail()

    @classmethod
    def tearDownClass(cls):
        print 'tearDownClass was executed!'
        super(VerifyTearDownClass, cls).tearDownClass()

    def test_123(self):
        pass
```
Result:
```python
============================= test session starts ==============================
platform linux2 -- Python 2.7.10, pytest-2.8.7, py-1.4.31, pluggy-0.3.1
rootdir: /home/andrii/workspace/robottelo, inifile: 
plugins: xdist-1.13.1, repeat-0.2
collected 33 items

workspace/robottelo/tests/foreman/cli/test_repository.py 2016-04-04 15:38:29 - robottelo - DEBUG - Running test VerifyTearDownClass/test_123
F
self = <tests.foreman.cli.test_repository.VerifyTearDownClass testMethod=test_123>

    def setUp(self):
        super(VerifyTearDownClass, self).setUp()
>       self.fail()
E       AssertionError: None

workspace/robottelo/tests/foreman/cli/test_repository.py:883: AssertionError
tearDownClass was executed!


=================================== FAILURES ===================================
_________________________ VerifyTearDownClass.test_123 _________________________

self = <tests.foreman.cli.test_repository.VerifyTearDownClass testMethod=test_123>

    def setUp(self):
        super(VerifyTearDownClass, self).setUp()
>       self.fail()
E       AssertionError: None

workspace/robottelo/tests/foreman/cli/test_repository.py:883: AssertionError
========= 32 tests deselected by '-k VerifyTearDownClass and test_123' =========
=================== 1 failed, 32 deselected in 2.82 seconds ====================

Process finished with exit code 0
```
As you can see, `tearDownClass was executed!` is there